pkg/virtctl/expose/expose.go

```diff
@@ -204,6 +204,7 @@ func (o *Command) RunE(cmd *cobra.Command, args []string) error {
 		ports = []v1.ServicePort{{Name: portName, Protocol: protocol, Port: port, TargetPort: targetPort}}
 	}
 
+	preferDualStack := v1.IPFamilyPolicyPreferDualStack
```

Comment:
Do ipFamilies still has any effect ? from the docs looks like if we set this both ipv4 and ipv6 are assigned ignoring ipFamilies.

https://kubernetes.io/docs/concepts/services-networking/dual-stack/#services

Reply:
I don't get that understanding from the example in https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/service/networking/dual-stack-preferred-ipfamilies-svc.yaml . It says (quoting from the link you've pasted):

"""
This Service specification explicitly defines IPv6 and IPv4 in .spec.ipFamilies as well as defining PreferDualStack in .spec.ipFamilyPolicy. When Kubernetes assigns an IPv6 and IPv4 address in .spec.ClusterIPs, .**spec.ClusterIP is set to the IPv6 address because that is the first element in the .spec.ClusterIPs array, overriding the default**.
"""